scripts/result_scanner/test/test_result_scanner.rb

```diff
@@ -0,0 +1,140 @@
+require 'minitest/autorun'
+require_relative '../result_scanner'
+
+# For now, you should just be able to `ruby test_result_scanner.rb`
+
+class TestResultScanner < MiniTest::Test
```

Comment:
In general yes.  In my experience RSpec suites get more unwieldy over time than MiniTest.  MiniTest is part of ruby as of 2.0 (I think, maybe earlier), not a separate dependency. (But it can and _should_ be added to `.Gemfile` as a specific dep.  if/when that happens.)  MiniTest can do spec-style tests too.